### ISSUE TITLE
How to open/close programmatically 

### ISSUE THREAD

**Issue by pythoneer** [bignerdranch/expandable-recycler-view#55]
What is the preferred way to programmatically  expand/collapse the views? I see no method to do so and i see no way to retrieve the views to fake click events on them. The overall design is very "closed" so i can't get access to anything that could help me, as far as i discovered this lib.

**Comment by paul-turner** [comment]
thanks @pythoneer, this functionality is something we are working on. Right now there isn't a public method available to do so. As a hack, you can modify [expandParent(ParentObject parentObject, int position)](https://github.com/bignerdranch/expandable-recycler-view/blob/master/expandablerecyclerview/src/main/java/com/bignerdranch/expandablerecyclerview/Adapter/ExpandableRecyclerAdapter.java#L314) to be public instead of private, but the actual change will require some reworking that we will get to very soon.

**Comment by pythoneer** [comment]
Excellent! :D .. i somehow feared that (i noticed that function in the first place). But i use your lib through gradle (compile 'com.bignerdranch.android:expandablerecyclerview:1.0.3'). Just for the record if other ppl needs that til you fixed that – i used reflections to do that.

```
            final int voteNum = 0; // number of element in the recycler view
            
            WsGetVotings_Response.WsVoting vote = currentVotings.get(voteNum);
            VotingObject vobj = new VotingObject(vote); // extends ParentObject

            Class<?>[] classArray = new Class<?>[2];
            classArray[0] = ParentObject.class;
            classArray[1] = int.class;

            Object[] parameters = new Object[2];
            parameters[0] = vobj;
            parameters[1] = voteNum;

            Method m = ExpandableRecyclerAdapter.class.getDeclaredMethod("expandParent", classArray);
            m.setAccessible(true);
            m.invoke(votingAdapter, parameters); // votingAdapter extends ExpandableRecyclerAdapter
```

i hope this helps someone in need :) ... and thank you very much for you quick help!

**Comment by paul-turner** [comment]
No problem, good workaround in the meantime. Hopefully we'll get the changes in soon to get rid of that reflection!

**Comment by dgreenhalgh** [comment]
Fixed in #67. Will be included in 2.0.0.
